### PR TITLE
[Jobs] Artifacts: preview content overflows

### PR DESCRIPTION
**Pull request by IvanVoznyi**

### CODE REVIEW

**Comment by mariana-furyk** [comment]
Screenshot from 2020-06-11 13-56-07

File: src/components/ArtifactsPreview/ArtifactsPreview.js [hunk 1]
```diff
@@ -20,6 +20,799 @@ const ArtifactsPreview = ({ artifact }) => {
   }, [])
 
   useEffect(() => {
+    // const res = {
```
Comment (mariana-furyk) [inline src/components/ArtifactsPreview/ArtifactsPreview.js:23]: remove comments

File: src/components/Details/details.scss [hunk 2]
```diff
@@ -225,6 +224,7 @@
       display: flex;
       flex: 1;
       flex-direction: column;
+      padding: 10px 21px;
```
Comment (mariana-furyk) [inline src/components/Details/details.scss:227]: add padding to preview content block not table

File: src/elements/DetailsInfoItem/DetailsInfoItem.js [hunk 3]
```diff
@@ -41,13 +41,15 @@ const DetailsInfoItem = ({
       </div>
     )
   } else if (!isEmpty(func)) {
+    let functionName = func?.name?.substr(1)
```
Comment (mariana-furyk) [inline src/elements/DetailsInfoItem/DetailsInfoItem.js:44]: Screenshot from 2020-06-15 15-04-29


And general you can use slice if you want to cut a string starting from some value

File: src/components/ArtifactsPreview/artifactaPreview.scss [hunk 4]
```diff
@@ -6,17 +6,28 @@
...
     color: $mulledWine;
 
-    &__row {
+    .table-header {
+      position: sticky;
+      top: -21px;
+      z-index: 2;
       display: flex;
+      width: fit-content;
+      min-width: 100%;
```
[the remark this answers is not in the document]
Comment (IvanVoznyi) [reply src/components/ArtifactsPreview/artifactaPreview.scss:17]: Could you clear this point up?

File: src/components/ArtifactsPreview/artifactaPreview.scss [hunk 5]
```diff
@@ -6,17 +6,28 @@
...
+    }
 
-      &:first-child {
-        position: sticky;
-        top: 0;
-        z-index: 2;
-        background-color: $white;
-      }
+    .table-body {
+      max-height: 500px;
```
[the remark this answers is not in the document]
Comment (IvanVoznyi) [reply src/components/ArtifactsPreview/artifactaPreview.scss:27]: min-height limit the height table body if the table is too large and the user can scroll through the table contents.

File: src/components/ArtifactsPreview/artifactaPreview.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (mariana-furyk) [reply src/components/ArtifactsPreview/artifactaPreview.scss:17]: Screenshot from 2020-06-16 09-59-09
Screenshot from 2020-06-16 09-59-18


Do you see that this styles are duplicated from table row?

**Pull request closed without merging**